src/App.js

```diff
@@ -24,40 +25,52 @@ export default class App extends Component {
 
   getLocationAndImage = async () => {
     const API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.citySearch}&format=json`
-    const response = await axios.get(API);
-    console.log(response.data[0]);
-    this.setState({ city: response.data[0] });
-    
-    // const API2 = `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_CITY_KEY}&center=${this.state.city.lat},${this.state.city.lon}&zoom=[1-18]&size=400x400&format=png&maptype=<MapType>roadmap</MapType>`
-    // const response2 = await axios.get(API2);
-
-  } 
+    try {
+      const response = await axios.get(API);
+      this.setState({
+        city: response.data[0],
+        error: '',
+      });
+    } catch (error) {
+      console.log(error);
+      this.setState({ 
+        error: `${error}`,
+        city: {},
+     })
+    }
+  }
 
   render() {
-    return(
+    return (
       <div id="formDiv">
         <Form>
           <Form.Label>Enter a City</Form.Label>
-          <Form.Control onChange={this.handleChange} value={this.state.citySearch} placeholder="Moscow" style={{width: '50%'}}/>
+          <Form.Control onChange={this.handleChange} value={this.state.citySearch} placeholder="Moscow" style={{ width: '50%' }} />
           <Button variant="primary" onClick={this.getLocationAndImage} onSubmit={this.getLocationAndImage}>
```

Comment:
Interesting that onClick and onSubmit are both on the button, let me know if you can get this working as I would also like to have the functionality of having the same information captured when the user hits "enter" or "return" on their keyboard.